scout/models/phenotype_term.py

```diff
@@ -65,7 +65,6 @@ def __init__(
     ):
         super(DiseaseTerm, self).__init__()
         self["disease_id"] = disease_id
-        self["_id"] = disease_id
```

Comment:
A way to get this in without the pain of having to stage reset of the table when updating would be to add another ID. I do agree it would have been better not to have two copies of the same, but... 😊